tool/flutter_ci_script_beta.sh

```diff
@@ -20,6 +20,9 @@ declare -ar PROJECT_NAMES=(
     "code_sharing/shared"
     "desktop_photo_search/fluent_ui"
     "desktop_photo_search/material"
+    // TODO(justinmc): To be enabled when its PR hits the master channel.
```

Comment:
// is not a comment in a bash script. Please change to #

Reply:
Big fail, thanks.